### PR TITLE
Small fixes on opening the dataset from DataView Window and setted the activated links color under Start and Help consistent to the Recent

### LINKED ISSUE #7017: Opening the dataset using the shortcut option under Start on the DataView window seems to not work
I have tried to open a dataset from the shortcut under `Start `on the DataView window and this seems to no longer work. Also, at the same time, it will be good to change the color when clicking on one of the options for opening a dataset or creating a new data frame as is happening with `Recent`.
image

### PR DESCRIPTION
**Pull request by N-thony**
Fixes #7017 
@africanmathsinitiative/developers this is ready for review. Thanks

### CODE REVIEW

**Comment by lloyddewit** [comment]
@rdstern Please can you test? thanks

**Comment by dannyparsons** [comment]
@ChrisMarsh82 I think this was just an unintentional change on the DataView?

It looks good to me.

**Review by lloyddewit — DISMISSED** [review]

**Comment by Patowhiz** [comment]
@N-thony I didn't know that there is PR on this control.
Could you also fix issue #7021. It is trivial and can just be included here to avoid conflicts.
I'm now going to close PR #7022.
Thanks.

**Comment by N-thony** [comment]
> @N-thony I didn't know that there is PR on this control. Could you also fix issue #7021. It is trivial and can just be included here to avoid conflicts. I'm now going to close PR #7022. Thanks.

@Patowhiz no problem. I will add it then.

**Review by dannyparsons — APPROVED** [review]
Changes look fine but someone please also test it

File: instat/ucrDataView.Designer.vb [hunk 1]
```diff
@@ -590,7 +590,7 @@ Partial Class ucrDataView
         '
         'linkHelpIntroduction
         '
-        Me.linkHelpIntroduction.ActiveLinkColor = System.Drawing.Color.Blue
+        Me.linkHelpIntroduction.ActiveLinkColor = System.Drawing.Color.Red
```
Comment (shadrackkibet) [inline instat/ucrDataView.Designer.vb:593]: I did not see any change as a result of this. The link remains `Blue`? Perhaps you could explain how I can check?

Comment (N-thony) [reply instat/ucrDataView.Designer.vb:593]: This determines the colors of the hyperlink when users click on,  as it is done under Recent

Comment (dannyparsons) [reply instat/ucrDataView.Designer.vb:593]: I think it's the colour as you click it, and then it changes back after clicking.

Comment (N-thony) [reply instat/ucrDataView.Designer.vb:593]: > 

yes

Comment (shadrackkibet) [reply instat/ucrDataView.Designer.vb:593]: Thanks, I see that it changes for the Recent links. However, I do not see the same thing when you click Recent links or Help links.

Comment (dannyparsons) [reply instat/ucrDataView.Designer.vb:593]: Let's make them consistent then.

Comment (N-thony) [reply instat/ucrDataView.Designer.vb:593]: It works fine on my side. It is just a small change when the users click on the Start or Help links.

Comment (shadrackkibet) [reply instat/ucrDataView.Designer.vb:593]: Apologies, I was not testing the most recent changes. Confirmed that it works now.

**Review by shadrackkibet — APPROVED** [review]

**Comment by Patowhiz** [comment]
@N-thony thanks.

**Pull request merged**